test/package.json

```diff
@@ -4,36 +4,45 @@
   "description": "Bedrock Tokenizer Tests",
   "scripts": {
     "test": "node --preserve-symlinks test.js test",
+    "test:debug": "node --preserve-symlinks test.js test --log-level debug",
     "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm test",
     "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcov npm test",
     "coverage-report": "nyc report"
   },
   "author": "",
   "dependencies": {
-    "bedrock": "^3.1.1",
-    "bedrock-account": "^3.0.0",
-    "bedrock-did-context": "^1.0.0",
-    "bedrock-express": "^3.2.0",
-    "bedrock-https-agent": "^2.0.0",
+    "bedrock": "^4.4.0",
+    "bedrock-account": "^5.0.0",
```

Comment:
Are bedrock-account, bedrock-passport, and bedrock-permission needed in the tests? If not, we should pare this down to what's needed.

Reply:
Will pare down.